### PR TITLE
[enhancement](load) add some profile items for load

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

Add some profile items to load, e.g. time consumed by sort, agg, download replica etc.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run beut

**Comment by dataroaring** [comment]
run beut

**Comment by dataroaring** [comment]
run beut

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

File: be/src/olap/memtable.cpp [hunk 1]
```diff
@@ -209,6 +209,8 @@ void MemTable::insert(const vectorized::Block* input_block, const std::vector<in
     for (int i = 0; i < num_rows; i++) {
         _row_in_blocks.emplace_back(new RowInBlock {cursor_in_mutableblock + i});
     }
+
+    _stat.raw_rows++;
```
Comment (morningman) [inline be/src/olap/memtable.cpp:213]: only add one row？

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.54 seconds
 stream load tsv:          443 seconds loaded 74807831229 Bytes, about 161 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          25.7 seconds inserted 10000000 Rows, about 389K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528054750_clickbench_pr_152085.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.65 seconds
 stream load tsv:          441 seconds loaded 74807831229 Bytes, about 161 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          25.6 seconds inserted 10000000 Rows, about 390K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528061024_clickbench_pr_152099.html

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.47 seconds
 stream load tsv:          445 seconds loaded 74807831229 Bytes, about 160 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          26.0 seconds inserted 10000000 Rows, about 384K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528110825_clickbench_pr_152163.html

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 36.14 seconds
 stream load tsv:          445 seconds loaded 74807831229 Bytes, about 160 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          25.9 seconds inserted 10000000 Rows, about 386K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528113508_clickbench_pr_152171.html

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.5 seconds
 stream load tsv:          445 seconds loaded 74807831229 Bytes, about 160 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          25.3 seconds inserted 10000000 Rows, about 395K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528114810_clickbench_pr_152176.html

**Comment by dataroaring** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.91 seconds
 stream load tsv:          454 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          26.6 seconds inserted 10000000 Rows, about 375K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528115838_clickbench_pr_152190.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.22 seconds
 stream load tsv:          441 seconds loaded 74807831229 Bytes, about 161 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          25.7 seconds inserted 10000000 Rows, about 389K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528121109_clickbench_pr_152198.html

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.96 seconds
 stream load tsv:          446 seconds loaded 74807831229 Bytes, about 159 MB/s
 stream load json:         23 seconds loaded 2358488459 Bytes, about 97 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          26.4 seconds inserted 10000000 Rows, about 378K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230528142307_clickbench_pr_152266.html

**Pull request merged**